PocketKit/Sources/Sync/Operations/SaveItemOperation.swift

```diff
@@ -41,7 +41,7 @@ class SaveItemOperation: SyncOperation {
             case ResponseCodeInterceptor.ResponseCodeError.invalidResponseCode(let response, _):
                 switch response?.statusCode {
                 case .some((500...)):
```

Comment:
Just to confirm we are disabling retry on an errors that are 500 or greater. Do we want to just fail only on 500 instead?

Reply:
@cyndichin yes that is corrrect.

Reply:
500+ are general server errors and not on the client https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#server_error_responses